import-service/handler.js

```diff
@@ -33,19 +33,38 @@ module.exports.importProductsFile = async event => {
 };
 
 module.exports.importFileParser = async event => {
-  const streams = event.Records.map(record => {
+  const sqs = new AWS.SQS();
+  const sns = new AWS.SNS();
+  const allData = [];
+  const streamsSent = event.Records.map(async record => {
     const params = {
       Bucket: bucket,
       Key: record.s3.object.key
     }
     const stream = s3.getObject(params).createReadStream().pipe(csv());
-    return new Promise((res, rej) => {
-        stream.on('data', d => console.log('data', d));
+    const dataSent = [];
+    const dataParsed = new Promise((res, rej) => {
+        stream.on('data', parsed => {
+          dataSent.push(sqs.sendMessage({
+              QueueUrl: process.env.SQS_URL,
+              MessageBody: JSON.stringify(parsed)
+            }).promise()
+          )
+          allData.push(parsed);
+        });
         stream.on('error', err => rej(err));
         stream.on('end', () => res())
-    });
+    })
+
+    await dataParsed;
+    await sns.publish({
```

Comment:
You need to move at least sns service (it better to move sns and sqs services) to product-service and send emails only after products are created

Reply:
Thanks
Yeah, I understand why it is better to do in this way, but I'm not going to do it until Sunday